src/tickets/dto/purchase-ticket.dto.ts

```diff
@@ -0,0 +1,8 @@
+import { BaseTicket } from '../interfaces/ticket.interface';
+
+export class PurchaseTicketDto implements BaseTicket {
+  userId: string;
+  quantity: number;
+  price: number;
```

Comment:
I don't think we need to re-declare those properties, they are already inherited from `BaseTicket`

Reply:
We can declare `PurchaseTicketDto` as `interface` instead of `class` 👍